src/screens/ExchangeModal.js

```diff
@@ -362,8 +363,9 @@ export default function ExchangeModal({
       });
     }
 
+    const outputInUSD = outputPriceValue * outputAmount;
     const gasPrice = selectedGasPrice?.txFee?.native?.value?.amount;
-    const cancelTransaction = await checkGasvInput(gasPrice, amountInUSD);
+    const cancelTransaction = await checkGasvInput(gasPrice, outputInUSD);
```

Comment:
the `outputPriceValue` is not guaranteed to exist either. We should ignore the check when a native price cannot be determined

Reply:
added a check for this